src/edu/csus/ecs/pc2/validator/customValidator/CustomValidatorSettings.java

```diff
@@ -102,7 +109,7 @@ public void setValidatorProgramName(String progName) {
      * @throws {@link RuntimeException} if the settings do not contain a valid Validator Interface mode setting
      */
     public String getCustomValidatorCommandLine() {
-        if (this.isUseClicsValidatorInterface()) {
```

Comment:
Agreed.  I was just following what the existing code that was there did.  It would look funny, seem odd and probably solicit more comments in the review if:

         if (this.isUseClicsValidatorInterface() || isUserInteractiveValidatorInterface()){

(one had `this `and one does not).
This is done in other places in this module as well (which doesn't make it right, but...).  All should be refactored at some point if we do it one place.  (Whoever wrote the module decided that "`this`" should be used for whatever reason.)
At this point, going through and removing all the "`this.`" is out of scope, no?